etc/avocado/avocado.conf

```diff
@@ -4,7 +4,8 @@ base_dir = /var/lib/avocado
 # You may override the specific test directory with test_dir
 test_dir = /usr/share/avocado/tests
 # You may override the specific test auxiliary data directory with data_dir
-data_dir = /var/lib/avocado/data
+#data_dir = /var/lib/avocado/data
+data_dir = /home/medic/avocado/data
```

Comment:
Sorry for force-push, I saw a failure, pushed the update and then realized you already reviewed it a bit...